corehq/apps/userreports/migrations/0019_ucrexpression_upstream_id.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.13 on 2022-05-09 14:31
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
nit: we've been moving towards isolating migrations when possible just to decrease the chances of dealing with a migration if the need to rollback a PR arises.

Reply:
Interesting - by isolating, do you mean I should spin this migration out into a separate PR, or just a separate commit?

Reply:
Yeah separating the migration into its own PR, though now I'm rethinking this. We don't appear to have this written in documentation anywhere, and I can see reasons to keep code and migration together. So it feels more like a case-by-case basis and I think this is fine to have them together given that you are only adding new functionality.

Reply:
Yeah, I can understand splitting out a migration that would migrate _data_ so it can be run separately. Migrations like that are likely hard to revert.